### DISCUSSION TITLE
sqlite的一个db文件可以同时作为元信息和数据存储吗？

### DISCUSSION THREAD

**Discussion by jihot2000** [juicedata/juicefs#2137] [Q&A] [answered]
几个可能有点“找茬”的问题：

1. sqlite的一个db文件可以同时作为元信息和数据存储吗？换句话说，我准备把元信息和数据都用sqlite存储，是不是只用一个db文件就可以？

2. 如果第一个问题是可以的话，那我一个db文件可以同时存放多个互相隔离的文件系统吗？如果第一个问题是否的话，一个db文件存元信息，一个db文件存数据，那我可以只使用这两个db文件存放多个互相隔离的文件系统吗？

3. 一个juicefs实现的文件系统，我在挂载的时候，可以只挂载里面的某个子文件夹吗？如果可以的话，就能把不同的子文件夹挂载到不同的位置上去，感觉也算变相地使用一套元信息和数据存储实现了多个独立的文件系统。

**Comment by davies** [comment, marked as answer]
同一个 db 文件 可以同时作为 元数据和数据存储。同时，但只能用于一个 文件系统。

挂载时，可以加上 --subdir 挂载一个子目录。

**Reply by jihot2000** [reply]
感谢解答，很强大
